src/lib/icu/meson.build

```diff
@@ -1,5 +1,6 @@
-icu_dep = dependency('icu-i18n', version: '>= 50', required: get_option('icu'))
-conf.set('HAVE_ICU', icu_dep.found())
+icu_i18n_dep = dependency('icu-i18n', version: '>= 50', required: get_option('icu'))
+icu_uc_dep = dependency('icu-uc', version: '>= 50', required: get_option('icu'))
+conf.set('HAVE_ICU', icu_i18n_dep.found())
```

Comment:
What if icu-i18n was found but not icu_uc?

Reply:
I can update to check both.

In theory, should be an unlikely scenario as `icu-i18n` has always used `icu-uc`. Though I guess there is no guarantee on what ICU will do in future.

Reply:
True, this *should* be both available, but since this is out of our control, and users may do stupid things (and some *will* do), let's be sure, and it's so simple to check both...